server/src/main/java/org/candlepin/resource/CloudRegistrationResource.java

```diff
@@ -55,16 +46,13 @@ public CloudRegistrationResource(I18n i18n, CloudRegistrationAuth cloudRegistrat
         this.cloudRegistrationAuth = Objects.requireNonNull(cloudRegistrationAuth);
     }
 
-    @POST
-    @Path("authorize")
-    @Produces(MediaType.TEXT_PLAIN)
-    @Consumes(MediaType.APPLICATION_JSON)
+    @Override
     @SecurityHole(noAuth = true)
-    public String authorize(CloudRegistrationDTO cloudRegDTO,
-        @Context Principal principal) {
-
+    public String cloudAuthorize(CloudRegistrationDTO cloudRegistrationDTO) {
```

Comment:
Why did the name of the method change here?

Reply:
I used cloudAuthorize as an operation id and it generates method names from the operation id. It has to be unique but we don't have such an endpoint yet so I can change it to authorize if you want.

Reply:
Nah, that's not critical. Just wanted understanding as to why it changed.